iaas/20240724.md

```diff
@@ -0,0 +1,85 @@
+# 2024-07-24
+
+## Participants
+
+- @gtema
+- @matofeder
+- @markus-hentsch
+- @flyersa
+- @janhorstmann
+- @scoopext
+- @o-otte
+- @kgube
+- @lindenb1
+- @ivnucko
+- @matusjenca2
+- @gndrmnn
+- @sbstnnmnn
+- @cah-patrickthiem
+- @mbuechse
+- @frosty-geek
+- @matfechner
+
+
+## Housekeeping
+
+- Report in this weeks community call: @markus-hentsch
+- Responsible for the minutes (writing and transfer to github): @gtema
+
+
+## Agenda
+
+### Need for Manila (@gtema)
+
+Typical Kubernetes usage pattern is to have a single PVC/PV shared between different pods. And since also typically K8 worker nodes are distributed across AZs it is required to have a storage that can be attached in different AZs. Usually (with OpenStack in mind) this is relying on Manila which we do not have as of now. Possibility to cross attach Cinder volumes across AZs (together with multiattach) is also not present by default.
```

Comment:
Not sure about this. I have not joined the IaaS call yesterday. Manila is aldready deployable with CephFS integration.